lib/src/components/NxDrawer/__tests__/NxDrawer.test.tsx

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { render, fireEvent, within, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import NxDrawer, { Props } from '../NxDrawer';
+import NxButton from '../../NxButton/NxButton';
+import useToggle from '../../../util/useToggle';
+
+describe('NxDrawer', function() {
+  const minimalProps: Props = {
+    open: true,
+    onClose: () => {},
+    children: 'Drawer Content'
+  };
+
+  const quickRender = rtlRender<React.ComponentPropsWithRef<typeof NxDrawer>>(NxDrawer, minimalProps);
```

Comment:
Why this extra complicated type?
```suggestion
  const quickRender = rtlRender<Props>(NxDrawer, minimalProps);
```

Reply:
> Why this extra complicated type?

Oh this was when we were testing ref before, forgot to revert it.